File: fizzbuzz.py
# fizzbuzzを実行するためのプログラム
# このプログラムを実行すると、1から100までの数字を出力する
# ただし、3の倍数のときはFizz、5の倍数のときはBuzz、3と5の倍数のときはFizzBuzzと出力する
i = 0
while i < 100:
    i += 1
    if i % 3 ==0 and i % 5  == 0:
        print("FizzBuzz")
    elif  i % 3 == 0:
        print("Fizz")
    elif  i % 5 == 0:
        print("Buzz")
    else:
        print(i)

File: javascript/fizzbuzz.js
//15で割り切れる時には"FizzBuzz"と出力する
//5で割り切れる時には"Buzz"と出力する
//3で割り切れる時には"Fizz"と出力する
//それ以外の時は数字をそのまま出力する
//1から100までの数字を出力する

for (var i = 1; i <= 100; i++) {
    if (i % 15 == 0) {
        console.log("FizzBuzz");
    } else if (i % 5 == 0) {
        console.log("Buzz");
    } else if (i % 3 == 0) {
        console.log("Fizz");
    } else {
        console.log(i);
    }
}